plugins/modules/cloud/opennebula/one_vm.py

```diff
@@ -1349,7 +1354,7 @@ def main():
         "api_url": {"required": False, "type": "str"},
         "api_username": {"required": False, "type": "str"},
         "api_password": {"required": False, "type": "str", "no_log": True},
-        "instance_ids": {"required": False, "aliases": ['ids'], "type": "list"},
+        "instance_ids": {"required": False, "aliases": ['ids'], "type": "list", "elements": "str"},
```

Comment:
Since the code uses `str(vm_id)` to add the IDs to a string in `get_vms_by_ids`, I'm not convinced that `elemenst=str` isn't wrong and will not break the module.

Reply:
Actually I just noticed in the RETURN doc it seems that field is actually an int, but what took me in the wrong path before is that the code also uses `int(vm_id)`

Reply:
commit coming up in few minutes